gitkraken-client/common-issues.md

```diff
@@ -8,6 +8,28 @@ taxonomy:
 ---
 
 ***
+### General Troubleshooting for GitKraken Client 9.4.0+
+
+In 9.4.0, GitKraken introduced the experimental feature for the [Git Executable](/gitkraken-client/experimental-features/#git-executable). As this is currently an experimental feature that our team is continuing to improve on, you can try adjusting your use of the Git Executable.
+
+Under `Preferences > Experimental`, first try adjusting the `Git Executable` version used. If you are using the “Bundled with GitKraken” version (comes pre-installed with GitKraken) try changing this to your own version of Git. If you do not have this installed, it can be downloaded [here](https://git-scm.com/download).
+
+The reason this may help is because the Bundled version is a minimal install of Git, which does not insclude all features.
```

Comment:
```suggestion
The reason this may help is because the Bundled version is a minimal install of Git, which does not include all features (specifically bash programs that may be used in Git Hooks).
```